pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -271,6 +272,119 @@ func updateApiConfig(s v1alpha1.TektonResultSpec) mf.Transformer {
 	}
 }
 
+// update the api container envs
```

Comment:
can you add more description here to show up that it also aupdates lockistack
that it is restricted for deployment and container with specific anmes